include/conc/concurrency.hpp

```diff
@@ -35,12 +35,17 @@ template <typename Mutex = std::mutex> class standard_policy {
             }
         }
     }
+
+    template <typename T>
+    constexpr static auto atomic_alignment_of() -> std::size_t {
```

Comment:
issue: i see this is never used anywhere

Reply:
It was a leftover experiment - removed.